ddcz/templates/users/sign_up.html

```diff
@@ -0,0 +1,131 @@
+{% extends "public.html" %}
+
+{% block styles %}
+<link rel="stylesheet" href="{{ reg_style }}">
+{% endblock %}
+
+{% block scripts %}
+<script type="text/javascript" src="{{ reg_script }}" async></script>
+{% endblock %}
+
+
+{% block content %}
+<h1 class="page-heading">Vstup do města Dračí Doupě</h1>
+        {% if sign_up_form.errors %}
+            {% for field in sign_up_form %}
+                {% for error in field.errors %}
```

Comment:
Also since field errors are not displayed together with fields, the field name should be printed out I think.

Reply:
I think this is just fine since the errors have those fields in them, like: "You use forbidden symbols in you nick."

But for those errors coming from Model I believe I don't have much of a control. They do not look very "human". But I'd skip that into the next version of the signing up.

Reply:
Ah, OK :)